templates/examples/constellation/text-processing/src/index.ts

```diff
@@ -0,0 +1,146 @@
+import _ from "lodash";
```

Comment:
Where is this being used? If I just missed it, is it something that can just be done with native JS API

https://youmightnotneed.com/lodash